packages/replay-next/components/sources/CurrentLineHighlight.tsx

```diff
@@ -82,15 +94,39 @@ function CurrentLineHighlightSuspends({ lineNumber, sourceId }: Props) {
           );
         }
       });
+      if (match != null) {
+        highlightColumnBegin = match.column;
+
+        columnBreakpointIndex = breakableColumnIndices.findIndex(
+          column => column === highlightColumnBegin
```

Comment:
I think it's possible to be paused at a location that isn't a breakable location...

Reply:
Oh?